D1673/P1673.md

```diff
@@ -6750,12 +6516,16 @@ private:
   Accessor @_nested-accessor_@; // exposition only
 
 public:
-  using reference     = /* see below */;
-  using element_type  = /* see below */;
+  using element_type = decltype(@_conj-if-needed_@(declval<Accessor::element_type>()));
+  using reference = element_type;
   using data_handle_type = typename Accessor::data_handle_type;
-  using offset_policy = /* see below */;
+  using offset_policy = conjugate_accessor<Accessor::offset_policy>;
 
-  constexpr conjugate_accessor(Accessor a);
+  constexpr conjugate_accessor(const Accessor& acc);
+  constexpr conjugate_accessor() = default;
+  constexpr conjugate_accessor(const scaled_accessor&) = default;
+  template<class OtherScalingFactor, class OtherNestedAccessor>
+    constexpr conjugate_accessor(const conjugate_accessor<OtherNestedAccessor>& other);
 
   constexpr reference access(data_handle_type p, size_t i) const
```

Comment:
This `access` function is conditionally `noexcept` (which is good).  Should we consider making the `access` function of `scaled_accessor` also conditionally `noexcept`?  Ditto for `offset`.

Reply:
yeah I think so.